src/main/javascript/src/views/assignment/MultipleAttemptsSetting.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <div>
+    <h4>Multiple attempts</h4>
+    <p class="grey--text text--darken-2 pb-0">
+      Decide if students should be able to submit the assignment more than once. 
+    </p>
+    <v-card outlined class="multiple-attempts-card">
+      <v-card-title :class="{ 'blue lighten-5': allowMultipleAttempts }">
+        <v-checkbox
+          v-model="allowMultipleAttempts"
+          class="mt-0"
+          label="Allow multiple attempts"
+          hide-details
+        ></v-checkbox>
+      </v-card-title>
+      <v-card-text v-if="allowMultipleAttempts" class="text--primary mx-0 px-5">
+        <div class="d-flex flex-wrap align-baseline">
+          <v-radio-group
+              v-model="allowInfiniteSubmissions"
+              column
+            >
+              <v-radio
+                label="blue"
+                color="blue"
+                class="mb-5"
+                :value="false"
+              >
+                <template v-slot:label>
+                    <div>A student is allowed up to <v-text-field
+                            outlined
+                            dense
+                            single-line
+                            hide-details
+                            :required="!allowInfiniteSubmissions"
+                            type="number"
+                            class="d-inline-block"
+                            :disabled="allowInfiniteSubmissions"
+                            v-model="numOfSubmissions"
+                            style="width:100px"
+                        ></v-text-field> attempts
+                    </div>
+                </template>
+              </v-radio>
+              <v-radio
+                label="A student is allowed an infinite number of attempts"
+                color="blue"
+                :value="true"
+              ></v-radio>
+          </v-radio-group>
+        </div>
+        <div class="mb-4"><v-divider class="mx-0 px-0"></v-divider></div>
+        
+        <div class="d-flex flex-column flex-wrap align-baseline">
+            <div class="mb-5">Minimum time between submissions: <v-text-field
+                    outlined
+                    dense
+                    single-line
+                    hide-details
+                    type="number"
+                    class="d-inline-block"
+                    v-model="hoursBetweenSubmissions"
+                    style="width:100px"
+                ></v-text-field> hours
+            </div>
+            <div class="mb-5">Keep the <v-select
+                :items="cumulativeOptions"
```

Comment:
I don't think this is captured in the ticket, but CUMULATIVE shouldn't be an option if the student is allowed infinite attempts.

Reply:
Fixed